src/app/screens/Calculator/index.js

```diff
@@ -0,0 +1,50 @@
+import React, {useState} from 'react';
+import {View, TextInput} from 'react-native';
+import {KeyboardAwareScrollView} from 'react-native-keyboard-aware-scroll-view';
+
+import styles from './styles';
+import Button from './components/Button';
+import { retrieveButtons } from './utils';
+
+const Calculator = () => {
+  //Variables y funcionesa
```

Comment:
This comments are not useful. Avoid commenting things that are clear in the code. Code should be written in a way that is self explanatory.

You should only be commenting things that are workarounds, patches or very particular and hard to understand by just reading.

Reply:
Do not make it dark, I understand what you are saying, I will omit the comments that do not add anything.